Angular-Migration.md

```diff
@@ -751,23 +751,41 @@ There may be some style updates required to make sure that the pages look exactl
 
 1. Ensure your frontend tests pass
 
+   Python:
    ```console
    python -m scripts.run_frontend_tests
    ```
+   
+   Docker:
+   ```console
+   make run_tests.frontend
```

Comment:
this command is getting 8885 success from 8886  and throwing this error below in picture 
![Uploading Screenshot from 2024-02-08 18-04-50.png…]()

Reply:
what about this @gp201 ? is this ok should be reviewed ?